.github/workflows/deploy-sage-account-web.yml

```diff
@@ -3,9 +3,11 @@ name: "Deploy SageAccountWeb"
 on:
   push:
     branches:
-      - SageAccountWeb-production
+      - production
+      - staging
```

Comment:
How do we discern `production` between SageAccountWeb and synapse-oauth-signin? Ideally we don't have to push both to prod at the same time

Reply:
Maybe `releases/SageAccountWeb/production` ? This branch naming convention lets us
- find all repo releases since they are prefixed by `releases/*`
- find all project-specific releases (same reason, `release/SageAccountWeb/*`)
- each project can have a project-specific release cadence and process (we could add `releases/SageAccountWeb/qa` if we needed to)